packages/wonder-blocks-modal/components/modal-launcher.js

```diff
@@ -6,6 +6,11 @@ import {View} from "wonder-blocks-core";
 import ModalLauncherPortal from "./modal-launcher-portal.js";
 import ModalBackdrop from "./modal-backdrop.js";
 
+// TODO(mdr): This should only match modal elements: StandardModal,
+//     TwoColumnModal, etc. But I had trouble getting the types to work for this
+//     union, especially when combined with `cloneElement`, so I'm punting.
+type ModalElement = React.Element<any>;
```

Comment:
It's possible that developers could write another component to return one of these three components as well and that should would.